samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/ui/CertificateViewerExamplesScreen.kt

```diff
@@ -0,0 +1,92 @@
+        ).map { X509Cert(it) }
+    )
+
+    val examples = mapOf(
+        "Key attestation certificate chain" to Cbor.encode(chainGkey.toDataItem()).toBase64Url(),
+        "Key Mint certificate chain" to Cbor.encode(chainGkey.toDataItem()).toBase64Url(), // TBA.
+        "Strong Box certificate chain" to Cbor.encode(chainGkey.toDataItem()).toBase64Url(), // TBA.
+        "MD IACA single certificate" to Cbor.encode(mlIaca.toDataItem()).toBase64Url(), // Single.
```

Comment:
I think it's sufficient to just have the first and the last here, can remove the ones with "// TBA.". We'll probably add more later, for exampler reader root certificates etc etc